CHANGELOG.md

```diff
@@ -26,6 +26,49 @@ Classify the change according to the following categories:
     ##### Removed
     ### Patches
 
+## Develop - 2022-06-27
+### Minor Updates 
+##### Changed
+`job/models.py` 
+- **Settings**
+    - Added **off_grid_flag**
+    - Changed **run_bau** to be nullable
+- **FinancialInputs**
+    - Added **offgrid_other_capital_costs**
+    - Added **offgrid_other_annual_costs**
+- **FinancialOutputs**
+    - Added **lifecycle_generation_tech_capital_costs**
+    - Added **lifecycle_storage_capital_costs**
+    - Added **lifecycle_om_costs_after_tax**
+    - Added **lifecycle_fuel_costs_after_tax**
+    - Added **lifecycle_chp_standby_cost_after_tax**
+    - Added **lifecycle_elecbill_after_tax**
+    - Added **lifecycle_production_incentive_after_tax**
+    - Added **lifecycle_offgrid_other_annual_costs_after_tax**
+    - Added **lifecycle_offgrid_other_capital_costs**
+    - Added **lifecycle_outage_cost**
+    - Added **lifecycle_MG_upgrade_and_fuel_cost**
+    - Added **replacements_future_cost_after_tax**
+    - Added **replacements_present_cost_after_tax**
+    - Added **offgrid_microgrid_lcoe_dollars_per_kwh**
+    - Changed **lifecycle_capital_costs_plus_om** and **lifecycle_om_costs_bau** field names to include before/after tax
+- **ElectricLoadOutputs**
+    - Added **offgrid_load_met_pct**
+    - Added **offgrid_annual_oper_res_required_series_kwh**
+    - Added **offgrid_annual_oper_res_provided_series_kwh**
+    - Added **offgrid_load_met_series_kw**
+- **ElectricTariffInputs**
+    - Changed all instances of `coincident_peak_load_active_timesteps` to `coincident_peak_load_active_time_steps`
+- **ElectricTariffOutputs**
+    - Changed field names to add suffixes denoting `before_tax` or `after_tax` values
+- **ElectricTariffInputs**
+    - Changed validation of this model to be conditional on **Settings.off_grid_flag** being False
+`job/run_jump_model.py` - Remove `run_uuid` key from input dictionary before running REopt to avoid downstream errors from Julia
```

Comment:
you removed `user_uuid`, right?

Reply:
Correct, it was causing errors from some downstream validations in REopt.jl